### ISSUE TITLE
Input and output for convertSolar2Lunar/convertLunar2Solar are not easy to use.

### ISSUE THREAD

**Issue by tvc12** [nguyen703/vnlunar#2]
Problem:
1. User must split parameters to correctly convert solar to lunar. Why do you use the `DateTime` type?
2.  Output does not demonstrate. Always remember what index describes day or month.

Suggestion:

1. You can use `DateTime` or an object instead of splitting parameters.
2. You can define `LunarDateTime` class to describe lunar DateTime instead of an array.
```dart

final DateTime inputDate = ... // 

/// class LunarDateTime {
///  public day; 
///  public month;
///  public year;
/// ... everything in here
/// }
final LunarDateTime lunarDateTime =  convertSolar2Lunar(inputDate);

// revert lunar -> solar
final DateTime solarDateTime = convertLunar2Solar(lunarDateTime);
```

**Comment by nguyen703** [comment]
Hi @tvc12,

Thank you for your contribution; I really appreciate it! :D

As mentioned in the README, this project is initially aimed at converting the JavaScript version of Dr. Ho Ngoc Duc into Dart code.

I realized there's a problem with using Lists instead of Classes, as you mentioned earlier. I've created a branch to reformat them and utilize Classes for easier use. However, I've been busy with another project, so I couldn't finalize it immediately.

I'll try to complete the conversion, but I can't promise an exact time.

Once again, I genuinely appreciate your contribution. Note that all PRs for new features are welcomed! 👍

Thank you,
Nguyen
